sdk/src/test/java/com/mapbox/maps/renderer/MapboxRendererTest.kt

```diff
@@ -98,9 +98,27 @@ internal abstract class MapboxRendererTest {
   }
 
   @Test
-  fun setMaximumFpsTest() {
+  fun setValidMaximumFpsTest() {
     mapboxRenderer.setMaximumFps(10)
-    verify { renderThread.setMaximumFps(10) }
+    verify(exactly = 1) { renderThread.setUserRefreshRate(10) }
+  }
+
+  @Test
+  fun setMoreThanMaximumFpsTest() {
+    mapboxRenderer.setMaximumFps(MapView.MAX_POSSIBLE_FPS.toInt() + 1)
```

Comment:
it's first not allowed value
```suggestion
    mapboxRenderer.setMaximumFps(MapView.MAX_POSSIBLE_FPS.toInt())
```